workflow/rocoto_funcs/jedivar.py

```diff
@@ -34,19 +38,13 @@ def jedivar(xmlFile, expdir):
     timedep=f'\n    <timedep><cyclestr offset="{starttime}">@Y@m@d@H@M00</cyclestr></timedep>'
   #
   COMROOT=os.getenv("COMROOT","COMROOT_NOT_DEFINED")
-  RUN='rrfs'
   NET=os.getenv("NET","NET_NOT_DEFINED")
   VERSION=os.getenv("VERSION","VERSION_NOT_DEFINED")
   dependencies=f'''
   <dependency>
   <and>{timedep}
     <taskdep task="prep_ic"/>
     <taskdep task="ioda_bufr"/>
-    <or>
```

Comment:
Instead of removing these ensemble file dependencies, can we add a variable for this? It can be empty for 3DVAR but should have ensemble file dependencies for EnVar,

Reply:
Good suggestion. We can introduce a switch to enforce ensemble dependencies for strict ensemble-related experiments.
Will do that in the next PR.

Reply:
Added an option “HYB_ENS_DEP" to force ensemble dependencies if needed (such as strictly testing the ensemble impacts).